sdk/mobile-center-crashes/src/test/java/com/microsoft/azure/mobile/crashes/WrapperSdkExceptionManagerTest.java

```diff
@@ -39,29 +42,40 @@ public void constructWrapperSdkExceptionManager() {
     public void loadWrapperExceptionData() throws Exception {
         PowerMockito.doThrow(new IOException()).when(StorageHelper.InternalStorage.class);
         StorageHelper.InternalStorage.readObject(any(File.class));
-        WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID());
+        assertNull(WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID()));
         PowerMockito.doThrow(new ClassNotFoundException()).when(StorageHelper.InternalStorage.class);
         StorageHelper.InternalStorage.readObject(any(File.class));
-        WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID());
-
-        WrapperSdkExceptionManager.loadWrapperExceptionData(null);
-
+        assertNull(WrapperSdkExceptionManager.loadWrapperExceptionData(UUID.randomUUID()));
+        assertNull(WrapperSdkExceptionManager.loadWrapperExceptionData(null));
```

Comment:
You should verify actual behavior of the method. Checking return value is not enough.